source/agora/consensus/protocol/Nominator.d

```diff
@@ -1127,6 +1160,9 @@ extern(D):
         header.updateSignature(multiSigCombine(sigs_to_add), validator_mask);
         log.trace("Updated block signature for block {}, mask: {}",
                 header.height, validator_mask);
+        log.dbg("{}: Number of signatures in header for block #{} is now {}",
+            __FUNCTION__, header.height, header.validators.setCount);
```

Comment:
Isn't it redundant with the `trace` message just before ?

Reply:
I'll combine the info into the Trace message.